packages/server/lib/modes/record.coffee

```diff
@@ -294,6 +304,15 @@ createRun = (options = {}) ->
       when 401
         recordKey = recordKey.slice(0, 5) + "..." + recordKey.slice(-5)
         errors.throw("DASHBOARD_RECORD_KEY_NOT_VALID", recordKey, projectId)
+      when 402
+        { code, payload } = err.error
+
+        numRuns = _.get(payload, "numRuns")
```

Comment:
it is a good idea to add a default value here - because the object might arrive without numRuns, and we don't want undefined